Sources/TuistKit/Commands/BuildCommand.swift

```diff
@@ -54,7 +54,7 @@ struct BuildCommand: ParsableCommand {
             absolutePath = FileHandler.shared.currentPath
         }
 
-        try BuildService().run(
```

Comment:
The convention that we've followed for services is that they only have one method and it's called `run`. Semantically `build` is more correct, but I'd keep `run` for consistency with the other services.